ORBIT/phases/install/cable_install/export.py

```diff
@@ -38,6 +39,7 @@ class ExportCableInstallation(InstallPhase):
         "landfall": {"trench_length": "km (optional)"},
         "export_cable_install_vessel": "str | dict",
         "export_cable_bury_vessel": "str | dict (optional)",
+        "array_cable_trench_vessel": "str (optional)",
```

Comment:
Should be "export_cable_trench_vessel".

Reply:
I'm in the file running the pre-commit cleanup scripts so I'll take care of this real quick.

Reply:
Oops, thanks!